java/src/main/java/org/yamlscript/yamlscript/LibYAMLScript.java

```diff
@@ -42,11 +46,18 @@ public static String path() throws RuntimeException
         if (path != null) return path;
 
         path = "/usr/local/lib" + pathDelimiter() + name;
-        if (new File(path).exists()) {
-            return path;
-        }
-
-        throw new RuntimeException("Shared library file " + name + " not found");
+        if (new File(path).exists()) return path;
+
+        path = System.getenv("HOME") + pathDelimiter() +
+               ".local/lib" + pathDelimiter() + name;
```

Comment:
This line wouldn't work in windows outside of wsl, but that only matters if windows is an intended supported platform (goes without saying), just thought I'd point it out